src/components/CardContainer/CardContainer.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import Card from "../Card/Card";
+import UniqueRandomArray from "unique-random-array";
+
+class CardContainer extends React.Component {
+    constructor(props) {
+        super(props)
+
+        this.state = {
+            dogs: []
+        }
+    }
+
+    componentDidMount() {
+        this.getDogs()
+    }
+
+    getDogs = () => {
+        fetch('http://localhost:3000/dogs', {
+            method: 'get'
+        })
+        .then((data) => data.json())
+        .then((res) => {
+            if(res.success) {
+                return res.data
+            } else {
+                return res.message
+            }
+        })
+            .then((res) => {
+                let random = UniqueRandomArray(res)
```

Comment:
The indentations is still really weird in this area - on my machine lines 31-34 are indented too far

Reply:
on it